src/app/page.js

```diff
@@ -10,5 +10,9 @@ export const metadata = {
 };
 
 export default function Page() {
-  return <div>Main</div>;
+  return (
+    <>
+      <TherapiesSection />
+    </>
```

Comment:
Do we need an empty tag here?

Reply:
nope, left from previous changes, removed